### PR TITLE
Users search bar

### LINKED ISSUE #27: Search form for users and organizations

### PR DESCRIPTION
**Pull request by aguspina**
Fixes #27

### CODE REVIEW

File: app/controllers/organizations_controller.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@ class OrganizationsController < ApplicationController
   include OrganizationsControllerTemplate
 
   def index
-    @organizations = Organization.accessible_as(current_user, :janitor)
+    @organizations = Organization.by_full_text(params[:q]).accessible_as(current_user, :janitor)
```
Comment (aguspina) [inline app/controllers/organizations_controller.rb]: I would like to filter at the end, but `accessible_as` returns an array 😢

File: app/models/concerns/with_search.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+module WithSearch
```
Comment (aguspina) [inline app/models/concerns/with_search.rb]: This is the same concern we have in `laboratory`, what do you think? @flbulgarelli

File: app/controllers/organizations_controller.rb (hunk 1 above)
Comment (flbulgarelli) [inline app/controllers/organizations_controller.rb]: Yes, you are right, but it ok.

File: app/models/concerns/with_search.rb (hunk 2 above)
Comment (flbulgarelli) [inline app/models/concerns/with_search.rb]: I know, but we don't have any place where to put that right now and it is not long enough to care about it.

**Pull request merged**